From a419a1b0e78603226fa231e26dacf95aa32bca55 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sat, 30 Apr 2016 22:46:41 +0100
Subject: Net::NNTP: Correct innd/nnrpd confusion in relation to Reader option

In the 'Reader' paragraph of the 'Constructor' section, the text has
nnrpd and innd the wrong way round

Bug: https://github.com/steve-m-hay/perl-libnet/pull/25
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=51962
Patch-Name: fixes/nntp_docs.diff
---
 cpan/libnet/lib/Net/NNTP.pm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cpan/libnet/lib/Net/NNTP.pm b/cpan/libnet/lib/Net/NNTP.pm
index 675b1e6a5..e043b3b76 100644
--- a/cpan/libnet/lib/Net/NNTP.pm
+++ b/cpan/libnet/lib/Net/NNTP.pm
@@ -845,10 +845,10 @@ NNTP server, a value of zero will cause all IO operations to block.
 B<Debug> - Enable the printing of debugging information to STDERR
 
 B<Reader> - If the remote server is INN then initially the connection
-will be to nnrpd, by default C<Net::NNTP> will issue a C<MODE READER> command
-so that the remote server becomes innd. If the C<Reader> option is given
+will be to innd, by default C<Net::NNTP> will issue a C<MODE READER> command
+so that the remote server becomes nnrpd. If the C<Reader> option is given
 with a value of zero, then this command will not be sent and the
-connection will be left talking to nnrpd.
+connection will be left talking to innd.
 
 B<LocalAddr> and B<LocalPort> - These parameters are passed directly
 to IO::Socket to allow binding the socket to a specific local address and port.
